src/Plugin/Field/FieldWidget/AvPortalWidget.php

```diff
@@ -100,101 +101,44 @@ public function formElement(FieldItemListInterface $items, $delta, array $elemen
       ];
     }
 
-    if ($source instanceof MediaAvPortalPhotoSource) {
+    if ($this->source instanceof MediaAvPortalSourceBase) {
```

Comment:
The source is always going to be an instance of this because above we rely on that to get the format. So this `if` is not needed.